GitUI/Editor/SearchCommitForm.cs

```diff
@@ -0,0 +1,122 @@
+﻿using GitCommands;
+
+namespace GitUI;
+
+public partial class SearchCommitForm : GitExtensionsDialog
```

Comment:
❔ The vast majority of the form names start with `Form`. 
```suggestion
public partial class FormSearchCommit : GitExtensionsDialog
```

Reply:
Almost no form in Editor has Form prefix. This was named in analogy with `FindAndReplaceForm`
Will rename though.